xml/System/DateTime.xml

```diff
@@ -6825,9 +6825,11 @@ The following example demonstrates the <xref:System.DateTime.Parse%28System.Stri
 ## Remarks  
  A single tick represents one hundred nanoseconds or one ten-millionth of a second. There are 10,000 ticks in a millisecond, or 10 million ticks in a second.  
   
- The value of this property represents the number of 100-nanosecond intervals that have elapsed since 12:00:00 midnight, January 1, 0001 (0:00:00 UTC on January 1, 0001, in the Gregorian calendar), which represents <xref:System.DateTime.MinValue?displayProperty=nameWithType>. It does not include the number of ticks that are attributable to leap seconds.  
+ The value of this property represents the number of 100-nanosecond intervals that have elapsed since 12:00:00 midnight, January 1, 0001 (0:00:00 on January 1, 0001, in the Gregorian calendar, in the current time zone), which represents <xref:System.DateTime.MinValue?displayProperty=nameWithType>.It does not include the number of ticks that are attributable to leap seconds.  
   
-   
+ As the starting instant of time is dependent on the local time zone, changing time zone using, for example, .ToUniversalTime() will potentially change the number of ticks, although the new value will represent the same instant in time.
```

Comment:
> As the starting instant of time is dependent on the local time zone [](start = 0, length = 68)

This is not true either, you can create a DateTime with Kind value Unspecified which not dependent at all on the local time zone